src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -29,6 +29,12 @@ export default class MutationInformationContainer extends React.Component<IMutat
         this.mergedMutations = this.mergeMutations(this.props.mutations);
     }
 
+    componentWillUpdate(nextProps:IMutationInformationContainerProps, nextState:{}) {
+        if (nextProps.mutations !== this.props.mutations) {
+            this.mergedMutations = this.mergeMutations(nextProps.mutations);
```

Comment:
see my slack comment about Reselect library, which I think is a more "idiomatic" way to handle the memoization of this derivation